1/noisifier/noisifier.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+import fnmatch 
+import numpy as np
+import librosa
+import soundfile as sf
+
+
+def read_audio(file_path, duration=None, sr=16000):
+    if file_path.endswith('.wav'):
+        return librosa.core.load(file_path, sr, duration=duration)
+    elif file_path.endswith('flac'):
+        return sf.read(file_path)
+    else:
+        raise ValueError(file_path + ': invalide format') 
+    
+
+def load_audio(file_path, size, sr=16000):
```

Comment:
Would be nice to have `size` as an optional argument and be able to infer file duration